Emoji Dictionary/requirements.txt

```diff
@@ -0,0 +1,3 @@
+libraries used : tkinter
+		 messagebox
+		 emoji
```

Comment:
just type the module, for example

```
tkinter
messagebox
emoji
```
so you can install by typing on cmd or terminal by
```
pip install -r requirements.txt
```